vipul/Gallery_System/index.html

```diff
@@ -0,0 +1,96 @@
+		<title>Photo Gallery | Index page</title>
+		<link rel="stylesheet" href="style.css">
+	</head>
+	<body>
+		<div class="container">
+			<div class="header">
+				<h1>
+					<a href="index.html">Photo Gallery</a>
+				</h1>
+			</div>
+			<div class="list">
+				<ul class="nav">
+					<li><a href="index.html" class="active">HOME</a></li>
+					<li class="sublist">
+						<a href="wild.html">CATEGORY</a>
+						<ul class="dropdown">
+							<li><a href="wild.html">Wildlife</a></li>
+							<li><a href="landscape.html">Landscape</a></li>
+							<li><a href="portrait.html">Portrait</a></li>
+							<li><a href="fashion.html">Fashion</a></li>
+							<li><a href="macro.html">Macro</a></li>
+							<li><a href="bird.html">Bird</a></li>
+							<li><a href="nature.html">Nature</a></li>
+							<li><a href="food.html">Food</a></li>
+						</ul>
+					</li>
+					<li><a href="services.html">SERVICES</a></li>
+					<li><a href="about.html">ABOUT</a></li>
+					<li><a href="contact.html">CONTACT</a></li>
+				</ul>
+			</div>
+		</div>
+		<div class="main-content">
+			<div class="grid-container" style="grid-auto-flow: column;">
+				<div class="gallery">
+					<a target="_self" href="wild.html">
+						<img src="images/tiger.jpeg" alt="Tiger">
+						<div class="centered">WildLife</div>
+					</a>
+				</div>
+				<div class="gallery">
+					<a target="_self" href="landscape.html">
+						<img src="images/landscape.jpg" alt="Landscape View">
+						<div class="centered">Landscape</div>
+					</a>
+				</div>
+				<div class="gallery">
+					<a target="_self" href="portrait.html">
+						<img src="images/portrait.jpg" alt="Portrait of a Girl">
+						<div class="centered">Portrait</div>
+					</a>
+				</div>
+				<div class="gallery">
+					<a target="_self" href="fashion.html">
+						<img src="images/fashion.jpg" alt="Image of men with pose">
+						<div class="centered">Fashion</div>
+					</a>
+				</div>
+				<div class="gallery">
+					<a target="_self" href="macro.html">
+						<img src="images/macro.jpg" alt="Picture of an Ant">
+						<div class="centered">Macro</div>
+					</a>
+				</div>
+				<div class="gallery">
+					<a target="_self" href="bird.html">
+						<img src="images/bird1.png" alt="Picture of a Bird">
+						<div class="centered">Bird</div>
+					</a>
+				</div>
+				<div class="gallery">
+					<a target="_self" href="nature.html">
+						<img src="images/nature.jpg" alt="View of Nature">
+						<div class="centered">Nature</div>
+					</a>
+				</div>
+				<div class="gallery">
+					<a target="_self" href="food.html">
+						<img src="images/food.jpg" alt="Picture of variety of dishes">
+						<div class="centered">Food</div>
+					</a>
+				</div>
+			</div>
+		</div>
+		<div class="footer">
+			<p>
+				Copyright 2020 All rights reserved | This is developed by <a href="https://www.facebook.com/vipul.suryawanshi" target="_blank">Vipul Suryawanshi.</a>
+			</p>
+		</div>
+	</body>	
+</html>	
```

Comment:
remove all extra spaces
Add more content in footer
Add contact us in footer
bad code indentation

Reply:
i will do all the changes